frontend/src/app/utils/testResults.ts

```diff
@@ -90,3 +91,10 @@ export const displayGuidance = (results: MultiplexResults) => {
     hasPositiveRsvResults(results)
   );
 };
+
+export const findResultByDiseaseNameForMultiplexResultInput = (
```

Comment:
suggestion: could we rename this to `findResultByDiseaseName`? Since it's typed for the `MultiplexResultInput` parameter I think we could shorten the name to make it more easily readable. Seems like *eventually* we might also want to find a more suitable name for `MultiplexResultInput` and `MULTIPLEX_DISEASES` since we're using it for more than just multiplex diseases now

Reply:
To me this reads as a chance to compress these two functions down into one, because they're both just reading an array of multiplex results, grabbing the result object based on the passed in disease name, and returning the test result. Any chance we could solve the problem by overloading these two functions together?

Reply:
Ah I missed seeing `getResultByDiseaseName` in this file, yeah I think it might be worth exploring consolidating these functions somehow but perhaps a follow up PR would be good for this, I'm fine with keeping `findResultByDiseaseNameForMultiplexResultInput` for this PR then

Reply:
i have something working in a local branch bc typescript function simplifications are like candy to me. @emyl3  i can put up a followup after this gets in

Reply:
@fzhao99 Nice!! Thank you -- will leave this as-is!